docs/source/index.rst

```diff
@@ -31,6 +31,11 @@ Installation
 
    pip install fsspec
 
+Not all included filesystems are usable by default without installing extra
+dependencies. For example to be able to access data in S3::
+
+   pip install fsspec[s3]
```

Comment:
Missing s3 in the list :)

Reply:
Please also check that this works - because s3fs depends on fsspec, fsspec[s3] would depend on itself.